java/src/main/java/com/cybersource/ws/client/MerchantConfig.java

```diff
@@ -321,13 +390,16 @@ public MerchantConfig(Properties _props, String _merchantID)
         useSignAndEncrypted = getBooleanProperty(merchantID, "useSignAndEncrypted", false);
         
         allowRetry  = getBooleanProperty(merchantID, "allowRetry", true);
```

Comment:
if (useHttpClient  && usePoolingHttpClient) {
            throw new ConfigException("both useHttpClient and usePoolingHttpClient cant be set to true, enable one of them");
        }

        if (useHttpClient || usePoolingHttpClient) {
            if(allowRetry) {
                numberOfRetries = getIntegerProperty(merchantID, "numberOfRetries", 5);
                if (numberOfRetries > 0)
                    retryInterval = getIntegerProperty(merchantID, "retryInterval", 5000);
                if (numberOfRetries < 1 || numberOfRetries > 5 || retryInterval < 0) {
                    throw new ConfigException("Invalid value of numberOfRetries and/or retryInterval");
                }
            }
        }